code/modules/clothing/spacesuits/rig.dm

```diff
@@ -120,6 +120,25 @@
 	species_restricted = list("exclude",VOX_SHAPED)
 	pressure_resistance = 40 * ONE_ATMOSPHERE
 	goliath_reinforce = TRUE
+	var/hidecount = 0
+
+/obj/item/clothing/head/helmet/space/rig/mining/attackby(var/obj/P, mob/user)
+	..()
+	if(istype(P,  /obj/item/asteroid/goliath_hide))
```

Comment:
We already have a system for reinforcing things with goliath plates, don't snowflake mining suits like this.